README.md

```diff
@@ -39,7 +39,9 @@ contacting us at security@hashicorp.com.
 
 An extensive quick start is viewable on the Consul website:
```

Comment:
It might be worth re-phrasing this now that we're linking to a few guides.

Reply:
Ok done.